vme/src/dilrun.cpp

```diff
@@ -224,6 +226,44 @@ void boot_global_dil(void)
    }
 }
 
+
+dilvar *getDilGlobalDilVar(const char *name, DilVarType_e type)
+{
+    if (g_global_dilvars[name] == nullptr)
+    {
+        // Create a placeholder variable for the global var
+
+        dilvar *v = new dilvar;
```

Comment:
This could be a good use case for unique_ptr because say the `new cNamelist` at 245 throws an exception the dilval would be freed then.